contents/handbook/people/team-structure/team-structure.md

```diff
@@ -1,10 +1,12 @@
 ---
-title: Team structure
+title: Team Structure
```

Comment:
```suggestion
title: Team structure
```

Changing back to sentence case

Reply:
But I'm not 100% sure about this. We're not consistent in `sidebar.json` here either.